<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="monthComboBox"/>
        </item>
        <item>
         <widget class="QSpinBox" name="yearSpinner">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>2000</number>
          </property>
          <property name="maximum">
           <number>2030</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Операций:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="transactionCountLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="transactionTableView">
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1301</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string notr="true">Файл</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string notr="true">Правка</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string notr="true">Вид</string>
    </property>
    <addaction name="actionCurrentMonth"/>
    <addaction name="separator"/>
    <addaction name="actionNextMonth"/>
    <addaction name="actionPrevMonth"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string notr="true">Сервис</string>
    </property>
    <addaction name="actionProfiles"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string notr="true">Окно</string>
    </property>
    <addaction name="actionAccountWindow"/>
    <addaction name="actionCardWindow"/>
    <addaction name="separator"/>
    <addaction name="actionContactWindow"/>
    <addaction name="actionCurrencyWindow"/>
    <addaction name="actionCategoryWindow"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string notr="true">Справка</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string notr="true">Выход</string>
   </property>
  </action>
  <action name="actionProfiles">
   <property name="text">
    <string notr="true">Профили...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+P</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Соединение...</string>
   </property>
  </action>
  <action name="actionContactWindow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Контакты...</string>
   </property>
  </action>
  <action name="actionCurrencyWindow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Валюты...</string>
   </property>
  </action>
  <action name="actionCategoryWindow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Категории...</string>
   </property>
  </action>
  <action name="actionAccountWindow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string notr="true">Счета...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionPrevMonth">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string notr="true">Предыдущий месяц</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Left</string>
   </property>
  </action>
  <action name="actionNextMonth">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string notr="true">Следующий месяц</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Right</string>
   </property>
  </action>
  <action name="actionCurrentMonth">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string notr="true">Текущий месяц</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Up</string>
   </property>
  </action>
  <action name="actionCardWindow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string notr="true">Карты...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
